features/step_definitions/preferences_steps.rb

```diff
@@ -140,3 +140,7 @@
   user.preference.adult = false
   user.preference.save
 end
+
+Given /^the user locale is "([^"]*)"$/ do |locale|
+  I18n.locale = locale
```

Comment:
Our translated emails use a setup like this:

https://github.com/otwcode/otwarchive/blob/5cd77dd6d6299fcd8b578115af460cad22ea3e78/app/mailers/user_mailer.rb#L61-L67

I think setting `I18n.locale` to test the abuse report confirmation email only works because it doesn't use that setup:

https://github.com/otwcode/otwarchive/blob/5cd77dd6d6299fcd8b578115af460cad22ea3e78/app/mailers/user_mailer.rb#L422-L431

So I'd rather not add this step for now. If we do add it, it should set up `preferred_locale` on the user's preferences.

Also in this case, we would be using `l(Time.now)` for a well-known format, so we can get away with not adding a specific test for it.

Reply:
Change to Cucumber tests reverted and replaced with a simpler, shorter and more explicit Rspec test: https://github.com/otwcode/otwarchive/pull/4168/commits/a2c477f5717bf4c8b03ad718aec1b7f019f556f1